kyma-serverless/fun/readme.md

```diff
@@ -0,0 +1,307 @@
+Power of serverless with SAP BTP, Kyma runtime. Dockerless and serverless
+==========
+
+### Pre-requisites
+
+1. Have your Free Tier SAP HANA Cloud database instance ready and up and running
+2. Have your Free Tier SAP Kyma, BTP runtime provisioned
+3. Prepare your Github account SSH key.  
+  * you may refer to this [blog post of mine](https://blogs.sap.com/2021/12/08/kyma-functions-with-gitrepository-source-with-ssh-authentication./) that describes how to enable SSH for your github account: 
+  * the source code needs to be stored in an *internet-facing* github repository thus other Github authentication methods are *strongly* discouraged
+  * furthermore, SSH may be the only way to access the source code in the github repositories with 2FA activated even the public ones.
+
+### Structure of the project  
+  * `fun-srv` folder contains the serverless source code
+  * `helm` folder contains the helm charts
+
+```
+📦fun
+ ┣ 📂fun-srv
+ ┃ ┣ 📜handler.js
+ ┃ ┗ 📜package.json
+ ┣ 📂helm
+ ┃ ┣ 📂fun-app
+ ┃ ┃ ┣ 📂templates
+ ┃ ┃ ┃ ┣ 📜NOTES.txt
+ ┃ ┃ ┃ ┣ 📜_helpers.tpl
+ ┃ ┃ ┃ ┣ 📜apirule.yaml
+ ┃ ┃ ┃ ┣ 📜configmap.yaml
+ ┃ ┃ ┃ ┣ 📜deployment.yaml
+ ┃ ┃ ┃ ┣ 📜resources.yaml
+ ┃ ┃ ┃ ┣ 📜service.yaml
+ ┃ ┃ ┃ ┗ 📜xs-app.yaml
+ ┃ ┃ ┣ 📜.helmignore
+ ┃ ┃ ┣ 📜Chart.yaml
+ ┃ ┃ ┗ 📜values.yaml
+ ┃ ┣ 📂fun-db
+ ┃ ┃ ┣ 📂templates
+ ┃ ┃ ┃ ┣ 📜NOTES.txt
+ ┃ ┃ ┃ ┣ 📜_helpers.tpl
+ ┃ ┃ ┃ ┣ 📜binding-hdi.yaml
+ ┃ ┃ ┃ ┣ 📜job.yaml
+ ┃ ┃ ┃ ┣ 📜service-hdi.yaml
+ ┃ ┃ ┃ ┗ 📜src.yaml
+ ┃ ┃ ┣ 📜.helmignore
+ ┃ ┃ ┣ 📜Chart.yaml
+ ┃ ┃ ┗ 📜values.yaml
+ ┃ ┗ 📂fun-srv
+ ┃ ┃ ┣ 📂templates
+ ┃ ┃ ┃ ┣ 📜NOTES.txt
+ ┃ ┃ ┃ ┣ 📜_helpers.tpl
+ ┃ ┃ ┃ ┣ 📜apirule.yaml
+ ┃ ┃ ┃ ┣ 📜binding-dest-x509.yaml
+ ┃ ┃ ┃ ┣ 📜binding-dest.yaml
+ ┃ ┃ ┃ ┣ 📜binding-uaa.yaml
+ ┃ ┃ ┃ ┣ 📜function.yaml
+ ┃ ┃ ┃ ┣ 📜service-dest.yaml
+ ┃ ┃ ┃ ┗ 📜service-uaa.yaml
+ ┃ ┃ ┣ 📜.helmignore
+ ┃ ┃ ┣ 📜Chart.yaml
+ ┃ ┃ ┗ 📜values.yaml
+ ┣ 📜Makefile
+ ┗ 📜readme.md
+
+```
+
+The fun project features a hdi deployer job that run once and a serverless backend service that runs forever...  
+The communication channel between the approuter frontend and the backend is via an approuter destination.
+
+### **Usage**:
+
+1. fork this public repository to yours or clone my repository and then push it yours
```

Comment:
Not sure what is meant here

Reply:
I changed the wording to reflect the above